<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_doctor_form</class>
 <widget class="QWidget" name="new_doctor_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="first_name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Фамилия</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="last_name"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Мед. организация</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="m_org_combo"/>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="ok_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancel_btn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>new_doctor_form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
